cmake/Doxyfile.in

```diff
@@ -791,25 +791,34 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = @CMAKE_SOURCE_DIR@/include \
-			 @CMAKE_SOURCE_DIR@/include/AP1roG \
-			 @CMAKE_SOURCE_DIR@/include/CISolver \
-			 @CMAKE_SOURCE_DIR@/include/FockSpace \ 
-			 @CMAKE_SOURCE_DIR@/include/HamiltonianBuilder \
-			 @CMAKE_SOURCE_DIR@/include/HamiltonianParameters \
-			 @CMAKE_SOURCE_DIR@/include/Operator \
-			 @CMAKE_SOURCE_DIR@/include/RDM \
-			 @CMAKE_SOURCE_DIR@/include/RHF \
-			 @CMAKE_SOURCE_DIR@/README.md \
-			 @CMAKE_SOURCE_DIR@/src \
-			 @CMAKE_SOURCE_DIR@/src/AP1roG \
-			 @CMAKE_SOURCE_DIR@/src/CISolver \
-			 @CMAKE_SOURCE_DIR@/src/FockSpace \ 
-			 @CMAKE_SOURCE_DIR@/src/HamiltonianBuilder \
-			 @CMAKE_SOURCE_DIR@/src/HamiltonianParameters \
-			 @CMAKE_SOURCE_DIR@/src/Operator \
-			 @CMAKE_SOURCE_DIR@/src/RDM \
-			 @CMAKE_SOURCE_DIR@/src/RHF
+INPUT      = "@CMAKE_SOURCE_DIR@/include" \
```

Comment:
So if I understand this correctly, every time a new subdirectory is added, we should also add it here?

Reply:
Yes.